grafana/scylla-overview.template.json

```diff
@@ -8,125 +8,1193 @@
+                    },
+                    {
+                        "class": "ops_panel",
+                        "gridPos": {
+                            "x": 12,
+                            "h": 6,
+                            "w": 12
+                        },
+                        "fieldConfig": {
+                            "overrides": [
+                                {
+                                    "matcher": {
+                                      "id": "byRegexp",
+                                      "options": "/method=\"[a-zA-Z]/"
+                                    },
+                                    "properties": [
+                                      {
+                                        "id": "custom.fillOpacity",
+                                        "value": 20
+                                      },
+                                      {
+                                        "id": "custom.drawStyle",
+                                        "value": "line"
+                                      },
+                                      {
+                                        "id": "unit",
+                                        "value": "short"
+                                      },
+                                      {
+                                        "id": "max",
+                                        "value": 1
+                                      },
+                                      {
+                                        "id": "custom.axisPlacement",
+                                        "value": "right"
+                                      },
+                                      {
+                                        "id": "custom.axisLabel",
+                                        "value": "Nemesis"
+                                      }
+                                    ]
+                                  }
+                            ]
+                        },
+                        "targets": [
+                            {
+                                "expr": "sum(rate(scylla_transport_requests_served{instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\"}[$__rate_interval])) by ([[by]])",
+                                "intervalFactor": 1,
+                                "legendFormat": "",
+                                "metric": "",
+                                "refId": "A",
+                                "step": 1
+                            },
+                            {
+                                "expr": "{__name__=~'nemesis(.*)(?:gauge)(.*)'}",
+                                "intervalFactor": 2,
+                                "refId": "B"
+                            }
+                        ],
+                        "title": "Requests Served per [[by]]",
+                        "type": "timeseries"
+                    },
+                    {
```

Comment:
I prefer to move this part to types.json i.e. create a new class for annotation list